runtime/runtime.go

```diff
@@ -94,6 +73,10 @@ type Params struct {
 	// where the contained document should be loaded.
 	Paths []string
 
+	// PluginDir is the path to a directory containing .builtin.so and .plugin.so shared object files that
+	// OPA can load dynamically to create custom builtins and plugins.
+	PluginDir string
```

Comment:
Can this go away? Applies to code below.

Reply:
Took it out.